tests/sentry/api/endpoints/test_group_attachments.py

```diff
@@ -29,12 +29,13 @@ def create_attachment(self, type=None, event_id=None):
     def path(self, types=None, event_ids=None):
         path = f"/api/0/issues/{self.group.id}/attachments/"
 
-        query = [("types", t) for t in types or ()]
-        if query:
-            path += "?" + urlencode(query)
+        typesQuery = [("types", t) for t in types or ()]
+        eventIdQuery = [("event_id", id) for id in event_ids or ()]
+        if typesQuery:
+            path += "?" + urlencode(typesQuery)
 
-        if event_ids:
-            path += "?event_ids=" + event_ids
+        if eventIdQuery:
+            path += "?" + urlencode(eventIdQuery)
```

Comment:
@DominikB2014 If `types` and `event_id` are present, then there is a `?` character in between, when it should be  the `&` character.

Reply:
that is true, I didn't account for that case because there are not tests that require both atm, but in the future it could be needed.
Good catch, I can update that 👍